### PR TITLE
docs(qa, smoke tests): Add Group Ex Pro/Search integration tests

### PR DESCRIPTION
**Pull request by podarok**
Fixes: https://openy.atlassian.net/browse/MAINTAIN-46

Extends: https://github.com/ymcatwincities/openy/wiki/Open-Y-Smoke-Tests-Index#open-y-features

Original: https://docs.google.com/spreadsheets/d/1yLUkMgJKK94hABy107_V-1AcJbRSSEf2s4wsQto1wfI/edit#gid=0&range=A60:I60

Original Issue, this PR is going to fix: https://openy.atlassian.net/browse/MAINTAIN-46

Make sure these boxes are checked before asking for review of your pull request - thank you!

If there is a new feature or this is a bug fix - use 9.x-2.x branch. We'll tag for release if the bug is critical asap or tag for release next bug fix release until critical issue arrived.

## Steps for review

- [ ] Check tests language.

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] Try to use Conventional commit messages accordingly to the standard https://www.conventionalcommits.org/en/v1.0.0/#specification
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/9.x-2.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/9.x-2.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3329/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3329/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3329/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3330/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3330/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3330/

File: modules/custom/openy_search/SMOKE_TESTS.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Welcome to Grid Content smoke tests documentation
```
Comment (hamrant) [inline modules/custom/openy_search/SMOKE_TESTS.md]: `Grid Content` is not related to this module, looks like copy/paste from different documentation

File: modules/custom/openy_search/SMOKE_TESTS.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+In order for Open Y Grid Content being tested in a short timeframe, please follow steps below
```
Comment (hamrant) [inline modules/custom/openy_search/SMOKE_TESTS.md]: Same here - `Open Y Grid Content`, should be `Open Y Search `

File: modules/custom/openy_search/SMOKE_TESTS.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Administrator / Anonymous
+
+### Steps
+
+1. Go to the homepage 
+2. Verify there is a search icon that opens search form when you click on it 
+3. Enter some keywords,  for example: ymca; Swim Lessons, etc
+4. Verify you see results on the page 
+5. Fin any landing page with some text on it and try to search by keywords from that landing page
```
Comment (hamrant) [inline modules/custom/openy_search/SMOKE_TESTS.md]: Typo - `Fin`, should be `Find`

**Review by hamrant — CHANGES_REQUESTED** [review]

File: modules/custom/openy_search/SMOKE_TESTS.md (hunk 1 above)
Comment (podarok) [reply modules/custom/openy_search/SMOKE_TESTS.md]: addressed https://github.com/ymcatwincities/openy/pull/2535/commits/f53eaddc5655b0974cf125f8d32468d4bb7788a4

File: modules/custom/openy_search/SMOKE_TESTS.md (hunk 2 above)
Comment (podarok) [reply modules/custom/openy_search/SMOKE_TESTS.md]: addressed https://github.com/ymcatwincities/openy/pull/2535/commits/f53eaddc5655b0974cf125f8d32468d4bb7788a4

File: modules/custom/openy_search/SMOKE_TESTS.md (hunk 3 above)
Comment (podarok) [reply modules/custom/openy_search/SMOKE_TESTS.md]: addressed https://github.com/ymcatwincities/openy/pull/2535/commits/e0550a87c09389cb078bd5e583ad31f77c18bd2c

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3331/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3331/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build3332/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/3332/

**Review by hamrant — APPROVED** [review]

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3331/

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build3332/

**Pull request merged**